addons/web_tour/__openerp__.py

```diff
@@ -13,7 +13,8 @@
     'depends': ['web'],
     'data': [
         'views/tour_templates.xml',
-        'views/tour_views.xml'
+        'views/tour_views.xml',
+        'security/ir.model.access.csv',
     ],
     'qweb': [
         "static/src/xml/*.xml",
```

Comment:
In fact, why do we want tours for public user ? Was it not decided it is for very first admin only ?